README.md

```diff
@@ -40,48 +57,49 @@ The entry in the application's `/server/datasources.json` will look like this:
+| Property                                                                                        | Type&nbsp;&nbsp; | Default | Description                                                                                                                                                                                                                                                                                                                       |
+| ----------------------------------------------------------------------------------------------- | ---------------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| allowExtendedOperators                                                                          | Boolean          | `false` | Set to `true` to enable using MongoDB operators such as `$currentDate`, `$inc`, `$max`, `$min`, `$mul`, `$rename`, `$setOnInsert`, `$set`, `$unset`, `$addToSet`, `$pop`, `$pullAll`, `$pull`, `$pushAll`, `$push`, and `$bit`.                                                                                                   |
+| enableGeoIndexing                                                                               | Boolean          | `false` | Set to `true` to enable 2d sphere indexing for model properties of type `GeoPoint`. This allows for indexed `near` queries.                                                                                                                                                                                                       |
+| lazyConnect                                                                                     | Boolean          | `false` | If set to `true`, the database instance will not be attached to the datasource and the connection is deferred. It will try to establish the connection automatically once users hit the endpoint. If the MongoDB server is offline, the app will start, however, the endpoints will not work.                                     |
+| disableDefaultSort                                                                              | Boolean          | `false` | Set to `true` to disable the default sorting                                                                                                                                                                                                                                                                                      |
+| behavior on `id` column, this will help performance using indexed columns available in MongoDB. |
```

Comment:
I think there's an extra `|` here that's putting this in a new row

Reply:
I think those weird format is causing by the formatting extension 😂